.gitattributes

```diff
@@ -0,0 +1,12 @@
+/.docker            export-ignore
+/.git*              export-ignore
+/.php-cs-fixer.php  export-ignore
+/*.svg              export-ignore
+/CHANGELOG.md       export-ignore
```

Comment:
maybe chanhelog and reamde no?

Reply:
Do you see any purpose to distribute them with the source? Is anyone looking for documentation there? Rather, they would look to github

Reply:
I could open my vendor, the package dir and read from the changelog file what has been changed?

Reply:
That's it, an user could want to do everything in a vendor, even copying the entire folder and running tests.
We don't want the user to do that, because allowing it would force us to distribute files that are not so useful in a vendor directory.

Reply:
I agree with you @garak

Reply:
ok you have convinced me